AnnoyancesFilter/sections/annoyances.txt

```diff
@@ -1773,6 +1773,7 @@ revouninstaller.com,logincasino.com,borsagundem.com,officelife.media,kod.ru,mast
 ! ###mc_embed_signup
 kosicednes.sk,tinyprojects.dev,cambio16.com,voxukraine.org,pnas.org,cachevalleydaily.com,bisnis.com,ilpost.it,bsblog.info,alcoparty.com.ua,codastory.com,informazionefiscale.it,kerch.fm,cinemablend.com,prostofinovo.ru,comss.ru,majorgeeks.com,buhgalter.com.ua,providr.com,24tv.ua,babylonbee.com,dexerto.com,football365.com,menrec.com,netdoktor.de,ottawamatters.com,retail-insider.com,walterfootball.com,wweek.com,zozhnik.ru###mc_embed_signup
 !
+forklog.com##.subscribe_blk
```

Comment:
we need to block `.subscribe_blk_desc` too

`forklog.com##div[class^="subscribe_blk"]`

Reply:
I don't know where you have found it. But I will add it anyway.

Reply:
![Screenshot 2021-04-15 at 16 13 09](https://user-images.githubusercontent.com/18536328/114875068-99f61b00-9e05-11eb-8673-f36aaeb091e3.png)